### DISCUSSION TITLE
Building without errors gives errorneous preview

### DISCUSSION THREAD

**Discussion by nath1as** [sveltejs/kit#10902] [Q&A]
Building my project gives no errors, but when running the preview (or deploying) the CSS isn't applied fully, the page is populated with css variables and html, but the individual classes aren't applied. There seems to be a mismatch between the generated files and files shown by vite preview. There are proper styles for a fragment of a second on first paint, and then it gets overridden by something. 

I think it gets built right, it gets cached right, but then something overrides it, no errors in build, preview, dev on CLI or browser...

I also get a console error from the sirv package which is a dependency of the sveltekit, it happens with yarn and bun, but not npm.

**Comment by bitdom8** [comment]
Same here
